cloud-deploy/apps/temperatures-route/.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
Any file that is not your application should not be in the PR. Please remove them from the PR.